tests/test_cmake_build/CMakeLists.txt

```diff
@@ -1,26 +1,22 @@
 add_custom_target(test_cmake_build)
 
-if(CMAKE_VERSION VERSION_LESS 3.1)
-  # 3.0 needed for interface library for subdirectory_target/installed_target
-  # 3.1 needed for cmake -E env for testing
-  return()
-endif()
 
-include(CMakeParseArguments)
 function(pybind11_add_build_test name)
   cmake_parse_arguments(ARG "INSTALL" "" "" ${ARGN})
 
-  set(build_options "-DCMAKE_PREFIX_PATH=${PROJECT_BINARY_DIR}/mock_install"
+  set(build_options "-DCMAKE_PREFIX_PATH=${pybind11_BINARY_DIR}/mock_install"
```

Comment:
This should never be executed with `${PROJECT_BINARY_DIR}` != `${pybind11_BINARY_DIR}`, imo, but yeah, it's just safer, I guess?

Reply:
The tests are now a subproject (as is supposed to be done in if you can run the subdirectory (tests) separately, which you can), so this is no longer true - PROJECT is the tests project. So this needs to be explicit.

Reply:
Ah, right, thanks!